### ISSUE TITLE
CUDA 9.2 compatibility 

### ISSUE THREAD

**Issue by HsuanTingLu** [lebedov/scikit-cuda#263]
## Problem

it seems that skcuda is calling a non existing method in the newer CUDA 9.2 library

error message: 
```
File "/home/user/anaconda2/lib/python2.7/site-packages/skcuda/fft.py", line 13, in <module>
    from . import cufft
 
File "/home/user/anaconda2/lib/python2.7/site-packages/skcuda/cufft.py", line 238, in <module>
    _libcufft.cufftSetCompatibilityMode.restype = int
 
File "/home/user/anaconda2/lib/python2.7/ctypes/__init__.py", line 379, in __getattr__
    func = self.__getitem__(name)
 
File "/home/user/anaconda2/lib/python2.7/ctypes/__init__.py", line 384, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

AttributeError: /usr/local/cuda/lib64/libcufft.so: undefined symbol: cufftSetCompatibilityMode
```

## Environment

- Ubuntu 16.04
- Python 2.7
- CUDA 9.2 installation through runfile
- PyCUDA 2017.1.1
- scikit-cuda 0.5.1

**Comment by lebedov** [comment]
Already fixed in latest revision on GitHub.

**Comment by HsuanTingLu** [comment]
note: solved in #248
